commands/undeploy_command.go

```diff
@@ -82,7 +82,9 @@ func (c *UndeployCommand) executeInternal(positionalArgs []string, dsHost string
 
 	force := GetBoolOpt(forceOpt, flags)
 	mtaID := positionalArgs[0]
-	if !force && !ui.Confirm("Really undeploy multi-target app %s? (y/n)", terminal.EntityNameColor(mtaID)) {
+	if !force && !ui.Confirm("Really undeploy multi-target app %s in org %s / space %s? (y/n)", terminal.EntityNameColor(mtaID),
+		terminal.EntityNameColor(cfTarget.Org.OrganizationFields.Name),
+		terminal.EntityNameColor(cfTarget.Space.SpaceFields.Name)) {
```

Comment:
There is no need to access the `*Fields` fields of the org and space.
The same can be achieved with `cfTarget.Org.Name` and with the space name accordingly.

Reply:
Done